### PR TITLE
Allprograms

### PR DESCRIPTION
**Pull request by Berenike-Kuhli**
Die Grundfunktion ist umgesetzt, die weiterführende Aufgabe würde ich in einem zweiten Angang später machen

### CODE REVIEW

File: src/components/ProgramRectangles.jsx [hunk 1]
```diff
@@ -1,6 +1,4 @@
-// import { ButtonPrimary } from "./buttons/ButtonPrimary";
-
-
+import ButtonPrimary from "./buttons/ButtonPrimary";
 
 export function ProgramLarge() {
```
Comment (SchmokStock) [inline src/components/ProgramRectangles.jsx]: We have pulled the current branch of allprograms. it is different than the code of this pull request here. Can oyu explain the difference between MediumPrograms and LargePrograms

File: src/pages/AllPrograms.jsx [hunk 2]
```diff
@@ -4,37 +4,37 @@ import { gql, useQuery } from "@apollo/client";
...
   return (
     <DefaultLayout>
       <h2 className="py-14 pl-4 text-left text-3xl font-bold text-light">
         Browse
       </h2>
-      {/* TODO make gradients alternate */}
-      <ProgramMedium>
-        {/* TODO add data from GraphQL */}
-        <p>Test</p>
+      {/* TODO make gradients alternate and get all the programs without hardcoding it here ... oder ist das zu kompliziert gedacht?*/}
```
Comment (SchmokStock) [inline src/pages/AllPrograms.jsx:27]: Instead connecting the programs of hygraph with gradients (or the other way around) you could keep the gradient order (Orange, Green, Blue, Orange... from up to down) and insert the data of programs despite of the color.

**Review by SchmokStock — APPROVED** [review]
heyya

File: src/components/ProgramRectangles.jsx (hunk 1 above)
Comment (Berenike-Kuhli) [reply src/components/ProgramRectangles.jsx]: ProgramLarge is a left over that is not used anymore

**Pull request merged**